cmd/zdb/zdb.c

```diff
@@ -6905,6 +6917,24 @@ dump_block_stats(spa_t *spa)
 				}
 			}
 		}
+		zdb_nicenum(mdstats->zb_count, csize,
+		    sizeof (csize));
+		zdb_nicenum(mdstats->zb_lsize, lsize,
+		    sizeof (lsize));
+		zdb_nicenum(mdstats->zb_psize, psize,
+		    sizeof (psize));
+		zdb_nicenum(mdstats->zb_asize, asize,
+		    sizeof (asize));
+		zdb_nicenum(mdstats->zb_asize / mdstats->zb_count, avg,
```

Comment:
In practice I doubt it can happen, but technically there's nothing preventing a divide by zero here (and the two cases below).

Reply:
I think both of these literally cannot happen or we couldn't have gotten to this code, since without metadata blocks it wouldn't have imported the pool enough to get here.